src/database/migrations/20180517100530_create_locales.js

```diff
@@ -0,0 +1,57 @@
+import localesJson from '../seeds/locales.json'
+
+export const up = async knex => {
+  await knex.schema.createTable('locales', table => {
+    table
+      .increments('id')
+      .unsigned()
+      .primary()
+    table.string('name', 50).notNull()
+    table
+      .string('code')
+      .unique()
+      .notNull()
+    table
+      .timestamp('createdAt')
+      .notNull()
+      .defaultTo(knex.fn.now())
+    table
+      .timestamp('updatedAt')
+      .notNull()
+      .defaultTo(knex.fn.now())
+  })
+
+  await knex.schema.createTable('projects_locales', table => {
+    table
+      .increments('id')
+      .unsigned()
+      .primary()
+    table
+      .integer('projectId')
+      .unsigned()
+      .references('id')
+      .inTable('projects')
+    table
+      .integer('localeId')
+      .unsigned()
+      .references('id')
+      .inTable('locales')
+    table
+      .timestamp('createdAt')
+      .notNull()
+      .defaultTo(knex.fn.now())
+    table.unique(['projectId', 'localeId'])
+  })
+
+  const localesArray = []
+
+  Object.entries(localesJson).map(([key, value]) => {
+    localesArray.push({ code: key, name: value })
+  })

```

Comment:
You could (should) simply have:

```js
const localesArray = Object.entries(localesJson).map(([key, value]) => ({ code: key, name: value })
```
Otherwise if you’re mutating something else, use `forEach` :)

Reply:
Also `localesArray`isn’t really the best naming, maybe `localesToSave` or something similar?